src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CaseRetrievalForCaseWorkerServiceTest.java

```diff
@@ -85,8 +89,10 @@ public void casesRetrievalESRequestException() throws IOException {
     public void casesRetrievalESRequest() throws IOException {
         List<SubmitEvent> submitEventList = Collections.singletonList(submitEvent);
         when(ccdClient.retrieveCasesElasticSearch(anyString(), anyString(), any())).thenReturn(submitEventList);
-        List<SubmitEvent> submitEventList1 = caseRetrievalForCaseWorkerService.casesRetrievalESRequest("1111", "authToken",
-                ccdRequest.getCaseDetails().getCaseTypeId(), new ArrayList<>(Collections.singleton("1")));
+        List<SubmitEvent> submitEventList1 = caseRetrievalForCaseWorkerService.casesRetrievalESRequest(
+                "1111", "authToken",
+                ccdRequest.getCaseDetails().getCaseTypeId(), new ArrayList<>(
```

Comment:
Push the new ArrayList to a new line. In general I think if a function call has too many parameters for one line, each parameter should be on a new line. There's been many cases of this throughout the PR. It's not the worst thing but it's much neater that way.